workflow-testing/src/main/java/com/squareup/workflow/testing/RenderTester.kt

```diff
@@ -350,6 +433,58 @@ interface RenderTester<PropsT, StateT, OutputT, RenderingT> {
    * handle a workflow or worker output or a rendering event.
    */
   fun render(block: (rendering: RenderingT) -> Unit = {}): RenderTestResult<PropsT, StateT, OutputT>
+
+  /**
+   * Describes a call to
+   * [RenderContext.renderChild][com.squareup.workflow.RenderContext.renderChild].
+   *
+   * @param workflow The child workflow that is being rendered.
+   * @param props The props value passed to `renderChild`.
+   * @param outputType The [KType] of the workflow's output type.
+   * @param renderingType The [KType] of the workflow's rendering type.
```

Comment:
Kdoc should mention that these types won't have any meaningful value when the workflow is created with inline builders, and link the bug